frontend/server/src/Controllers/Problem.php

```diff
@@ -5978,6 +5978,7 @@ public static function getLibinteractiveGenForTypeScript(
      * @omegaup-request-param mixed $order_by
      * @omegaup-request-param int|null $page
      * @omegaup-request-param null|string $programming_languages
+     * @omegaup-request-param null|string $quality
```

Comment:
```suggestion
     * @omegaup-request-param null|'onlyQualityProblems'|'all' $quality
```

igual en todos los demás lugares donde esto se está agregando.

Reply:
Listo :smile: